examples/todo/ui/todo.slint

```diff
@@ -17,20 +18,25 @@ MainWindow := Window {
     callback popup_confirmed;
     callback show_confirm_popup;
     show_confirm_popup => { confirm_popup.show(); }
-    
+
     confirm_popup := PopupWindow {
         x: 40px;
         y: 100px;
-        Rectangle { height:100%; width: 100%; background: root.background; }
-        VerticalBox {
-           Text {
-               text: "Some items are not done, are you sure you wish to quit?";
-            }
-            HorizontalLayout {
-                 Button { text: "Yes"; clicked => { popup_confirmed(); } }
-                 Button { text: "No"; }
+        width: min(confirm_popup_layout.preferred-width, root.width - 80px);
+        Rectangle {
+            background: root.background;
+            border-color: confirm_popup_text.color;
+            border-width: 1px;
```

Comment:
I think the border should be done by the PopupWindow.
the macos native style already does that. This is why i didn't add one.
on my system this overrides the border with the correct native color.

Reply:
on X11 there is no border at all